operator/cmd/loki-broker/main.go

```diff
@@ -39,8 +39,9 @@ func (c *config) registerFlags(f *flag.FlagSet) {
 	// Feature flags
 	c.featureFlags = configv1.FeatureGates{}
 	c.featureFlags.OpenShift = configv1.OpenShiftFeatureGates{}
```

Comment:
These two lines should be redundant as both values are not pointer values and so are already initialized with their zero value.